### PR TITLE
:globe_with_meridians: Add Spanish translation

### PR DESCRIPTION
**Pull request by dev-oswld**
Final results for Spanish translation.
Issue: #63

### CODE REVIEW

**Comment by kefranabg** [comment]
Awesome @dev-oswld  thanks for this, we'll do a review ASAP!

**Comment by mbernardeau** [comment]
Could you add the new file to `.allcontributorssrc` ? 

https://github.com/zenika-open-source/open-source-promotion-cheat-sheet/blob/master/.all-contributorsrc#L5

File: README-es.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+> Identifica las fortalezas de tu proyecto y asegúrate de que los visitantes las vean primero.
+
+</p>
+</details>
+
+<details>
+<summary>✨ Proporciona una demostración de tu proyecto</summary>
+<p>
+
+> los visitantes desean comprender rápidamente el propósito de tu proyecto, cómo funciona y cómo usarlo. Proporciona una demostración, es la mejor manera de complacer a los usuarios. Podría ser:
```
Comment (Zenigata) [inline README-es.md]:
```suggestion
> Los visitantes desean comprender rápidamente el propósito de tu proyecto, cómo funciona y cómo usarlo. Proporciona una demostración es la mejor manera de complacer a los usuarios. Podría ser:
```

File: README-es.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+<summary>👌 Instalación/Manejo debe ser súper fácil</summary>
+<p>
+
+> Probablemente perderás visitantes si tu proyecto no es fácil de usar.
+
+</p>
+</details>
+
+<details>
+<summary>📘 Crea una documentación limpia y bien estructurada.</summary>
```
Comment (Zenigata) [inline README-es.md]:
```suggestion
<summary>📘 Crea una documentación limpia y bien estructurada</summary>
```

File: README-es.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+<summary>⭐ Haz que tu proyecto sea confiable antes de publicarlo en las redes sociales</summary>
+<p>
+
+> La mayoría de los visitantes comprobarán cuántas estrellas tiene tu proyecto antes de considerar usarlo. Una cantidad mínima de estrellas hace que tu proyecto sea más confiable que un proyecto con cero estrellas. Es por eso que debes pedir a las personas que conoces que apoyen tu proyecto antes de hacer un anuncio público en las redes sociales.
+
+</p>
+</details>
+
+<details>
+<summary>↗️ Comparte tu proyecto en redes sociales y plataformas especializadas.</summary>
```
Comment (Zenigata) [inline README-es.md]:
```suggestion
<summary>↗️ Comparte tu proyecto en redes sociales y plataformas especializadas</summary>
```

File: README-es.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+> No dejes problemas pendientes sin respuesta. Sé amable con las personas que se tomaron el tiempo para dejar una sugerencia. 😉
+
+</p>
+</details>
+
+<details>
+<summary>🙏 Invita a otros usuarios a contribuir</summary>
+<p>
+
+> Un proyecto saludable es un proyecto con una comunidad y contribuidores activos. Informa a tus usuarios que necesita ayuda etiquetando algunos problemas con las etiquetas de  `contribution welcome` o `good first issue`. [Ver más etiquetas de github](https://help.github.com/en/articles/about-labels).
```
Comment (Zenigata) [inline README-es.md]:
```suggestion
> Un proyecto saludable es un proyecto con una comunidad y contribuidores activos. Informa a tus usuarios que necesita ayuda etiquetando algunos problemas con las etiquetas de `contribution welcome` o `good first issue`. [Ver más etiquetas de github](https://help.github.com/es/github/managing-your-work-on-github/about-labels).
```

File: README-es.md [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+
+</p>
+</details>
+
+<details>
+<summary>🏆 Recompensas a los contribuidores</summary>
+<p>
+
+> ¡Sé amable con las personas que te ayudaron! Algunos proyectos de código abierto como [gatsby](https://github.com/gatsbyjs/gatsby) premian a los contribuidores. Si no puedes pagar eso, haz una publicación (en twitter u otras plataformas) sobre la contribución y menciona al autor ([este es un ejemplo de agradecimiento público](https://twitter.com/FranckAbgrall/status/1139470547492978688 )). Abre una sección de `Contribuidores` en tu archivo README para agradecerles públicamente o mostrarlos en la documentación de tu proyecto o sitio web. Aquí hay unos ejemplos:
```
Comment (Zenigata) [inline README-es.md]:
```suggestion
> ¡Sé amable con las personas que te ayudaron! Algunos proyectos de código abierto como [gatsby](https://github.com/gatsbyjs/gatsby) premian a los contribuidores. Si no puedes pagar eso, haz una publicación (en twitter u otras plataformas) sobre la contribución y menciona al autor ([este es un ejemplo de agradecimiento público](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Abre una sección de `Contribuidores` en tu archivo README para agradecerles públicamente o mostrarlos en la documentación de tu proyecto o sitio web. Aquí hay unos ejemplos:
```

File: README-es.md [hunk 6]
```diff
@@ -0,0 +1,286 @@
...
+> Los visitantes confiarán en tu proyecto si ven casos de uso concretos e historias de éxito, por ejemplo, [la gallería vuepress](https://vuepress.gallery/).
+
+</p>
+</details>
+
+<br />
+
+## 🙏 Muestra tu apoyo
+
+⭐️ a este repositorio ¡si esta hoja de referencia te ayudo!
```
Comment (Zenigata) [inline README-es.md]: **te ayuda** or **te ayudó**?

File: README-es.md (hunk 1 above)
Comment (dev-oswld) [reply README-es.md]: Thanks for the observation

File: README-es.md (hunk 6 above)
Comment (dev-oswld) [reply README-es.md]: Small details matter

File: README-es.md [hunk 7]
```diff
@@ -254,7 +254,7 @@ Lista de lenguajes disponibles:
 
 ## 🙏 Muestra tu apoyo
 
-⭐️ a este repositorio ¡si esta hoja de referencia te ayudo!
+⭐️ a este repositorio ¡si esta hoja de referencia te ayudó!
```
Comment (dev-oswld) [inline README-es.md]: Small details matter.

File: README-es.md [hunk 8]
```diff
@@ -0,0 +1,286 @@
+<p align="center">
+    <img alt="oss image" src="./imgs/zoss-logo.svg" height="200px" width="200px">
+</p>
+
+<p align="center">
+  <a href="https://twitter.com/intent/tweet?text=How%20to%20promote%20your%20open-source%20projects%20@ZenikaOSS&url=https://github.com/zenika-open-source/open-source-promotion-cheat-sheet&hashtags=OpenSource,CheatSheet">
+    <img alt="tweet" src="https://img.shields.io/twitter/url/https/twitter?label=Share%20on%20twitter&style=social" target="_blank" />
```
Comment (kefranabg) [inline README-es.md]: Could you trasnlate "Share on twitter" ?

File: README-es.md [hunk 9]
```diff
@@ -0,0 +1,286 @@
...
+</p>
+
+<h1 align="center">Hoja de referencia de promoción al código abierto</h1>
+
+
+> Esta hoja de referencia resume los pasos importantes que debes seguir para promover tu proyecto de código abierto en óptimas condiciones. Puedes hacer clic en los elementos para expandir y obtener más información de un tema.
+
+Lista de lenguajes disponibles:
+
+- [Inglés](./README.md)
```
Comment (kefranabg) [inline README-es.md]: No need to translate these countries in Spanish (except Spanish) 😉

File: README-es.md (hunk 8 above)
Comment (dev-oswld) [reply README-es.md]: Of course!

File: README-es.md (hunk 9 above)
Comment (dev-oswld) [reply README-es.md]: Okok

**Comment by kefranabg** [comment]
@all-contributors please add @dev-oswld for translation

**Comment by dev-oswld** [comment]
:fireworks: yeah!

**Pull request merged**